### ISSUE TITLE
Plastic Gun

### ISSUE THREAD

**Issue by 5hart** [studio-minus/ppg-bugs-and-requests#994]
**Feature description**
Some type of pistol that is made out of plastic.

**Feature purpose**
It's lighter, doesn't get detected under metal detectors, and doesn't get attracted or repulsed by the electromagnet.

**Additional context**
I believe the 9mm says it's plastic framed, but still gets detected by the metal detector.

**Comment by WWWOWWWPL** [comment]
i believe the vanilla 9mm pistol is based off the IRL Glock 17 which still has metal in it so metal detectors can easily identify it

**Comment by 5hart** [comment]
> i believe the vanilla 9mm pistol is based off the IRL Glock 17 which still has metal in it so metal detectors can easily identify it

Yeah, I figured. Do you still think a 100% plastic gun is possible?

**Comment by DindinYT37** [comment]
> > i believe the vanilla 9mm pistol is based off the IRL Glock 17 which still has metal in it so metal detectors can easily identify it
> 
> Yeah, I figured. Do you still think a 100% plastic gun is possible?

Probably, guns can be 3d-printed nowadays (other than the barrel, I think). Well it's definitely possible in ppg, I'd say it's just a weak version of a normal gun.

**Comment by 5hart** [comment]
> > > i believe the vanilla 9mm pistol is based off the IRL Glock 17 which still has metal in it so metal detectors can easily identify it
> > 
> > 
> > Yeah, I figured. Do you still think a 100% plastic gun is possible?
> 
> Probably, guns can be 3d-printed nowadays (other than the barrel, I think). Well it's definitely possible in ppg, I'd say it's just a weak version of a normal gun.

Yeah, chances are it can be weaker, but I do love the realism and physics this game has!

**Comment by wordpass1** [comment]
Epic suggestion! Could be used for stealth missions where your humans inevitably die.
